plugins/org.jkiss.dbeaver.ext.mysql/src/org/jkiss/dbeaver/ext/mysql/model/MySQLUser.java

```diff
@@ -72,7 +76,9 @@ public MySQLUser(MySQLDataSource dataSource, ResultSet resultSet) {
         if (resultSet != null) {
             this.persisted = true;
             this.userName = JDBCUtils.safeGetString(resultSet, "user");
+            persistedUserName = userName;
```

Comment:
Add `this.` for consistency

Reply:
To be quite honest, it's tempting to delete `this` qualifier from other statements rather than add it to new ones.